sys/net/routing/rpl/rpl_control_messages.c

```diff
@@ -539,19 +532,6 @@ void rpl_recv_DIO(void)
             return;
         }
     }
-    else if (my_inst == NULL) {
-        DEBUGF("Not joined an instance yet\n");
-    }
```

Comment:
this information cannot be retrieved here, because this part of the code handles the instances and not the dodags. The implementation will create any instance it encounters, if it did exist before, it will skip the creation process. This is all to this section of the code, therefore it was kind of too complex before..